DocumentService/Controllers/DocumentsAnonymousController.cs

```diff
@@ -0,0 +1,151 @@
+﻿namespace DocumentService.Controllers
+{
+    using DocumentService.Authorization;
+    using DocumentService.Azure;
+    using DocumentService.Models;
+    using DocumentService.Repositories;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Identity.Web.Resource;
+    using System;
+    using System.Collections.Generic;
+
+    [Authorize]
+    [ApiController]
+    [Route("api/")]
+
+    [RequiredScope(RequiredScopesConfigurationKey = ScopePolicy.ReadWritePermission)]
```

Comment:
This line should be removed in this class.

Reply:
Oh I missed this one.
Many thanks